lib/aptible/cli/subcommands/operation.rb

```diff
@@ -16,6 +16,24 @@ def self.included(thor)
               CLI.logger.info m
               o.update!(cancelled: true)
             end
+
+            desc 'operation:connect OPERATION_ID',
+                 'Follow logs of a running operation'
+            define_method 'operation:connect' do |operation_id|
+              o = Aptible::Api::Operation.find(operation_id, token: fetch_token)
+              raise "Operation ##{operation_id} not found" if o.nil?
+
+              if %w(failed succeeded).include? o.status
```

Comment:
not sure if this will behave in a `queued` status

Reply:
Yes, it will still connect if the operation is queued.